### PR TITLE
Added a clear method that deletes all the values in a session

### PR DESCRIPTION
**Pull request by jshrake**
This pull request implements the functionality discussed in issue #3.

### CODE REVIEW

File: sessions.go [hunk 1]
```diff
@@ -117,6 +119,12 @@ func (s *session) Delete(key interface{}) {
 	s.written = true
 }
 
+func (s *session) Clear() {
+	for key := range s.Session().Values {
+		s.Delete(key)
```
Comment (jshrake) [inline sessions.go:124]: I contemplated repeating the code from the Delete method and then setting `s.written = true` only once. The current implementation has the advantage of code reuse and that `s.written` is only updated if `Values` is non-empty.

**Comment by codegangsta** [comment]
Very cool. I will take a look at this soon

File: sessions.go (hunk 1 above)
Comment (codegangsta) [inline sessions.go:124]: I like this approach for now. I doubt we will be bottlenecking on iteration time here :)

**Comment by codegangsta** [comment]
Nice work! Thanks for putting this together

**Comment by ghost** [comment]
Nice! Perhaps add it in the example in the readme and close this issue?

**Pull request merged**